src/components/ModalContainer/index.js

```diff
@@ -17,6 +17,9 @@ const customStyles = {
     borderRadius: '5px',
     textAlign: 'center',
     maxHeight: '100vh',
+    maxWidth: '100vh',
```

Comment:
This also affects to all other modals! You could add separate class for image preview container, so that profile page and bot modals would not be affected.

Reply:
Whoops, so it does. I replaced it with a new modal-container and the styles are changed to correspond more closely with the desired result.